### PR TITLE
Minify raw svgs

### PR DESCRIPTION
**Pull request by JackHowa**
- Small optimization for svgs
- Bigger question though ... are we using the raw svgs anywhere? I don't see them in the blocks or engine theme sdk ... maybe custom blocks? If not, may as well not publish at all the raw svgs https://github.com/WPMedia/engine-theme-sdk/pull/85/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L12
- See more in #82

### CODE REVIEW

File: svg/arc-placeholder-logo.svg [hunk 1]
```diff
@@ -1,31 +1 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (brentmiller1973) [inline svg/arc-placeholder-logo.svg:1]: Nice (removing the xml doctype)

**Comment by jgrosspietsch** [comment]
At his point I've completely forgotten my reasoning for including the SVGs alongside the icon components.

**Review by jgrosspietsch — APPROVED** [review]

**Review by brentmiller1973 — CHANGES_REQUESTED** [review]
Jack this is good first pass!  However I think now might be the opportunity to remove the height and width attributes to allow them to be more responsive.  The height and width should be handled via CSS/SCSS.  The only attributes that should be in the svg parent tag are xmlns, viewBox and in some cases, xmlns:xlink

**Comment by JackHowa** [comment]
@visionbegin cool good idea ... done via https://github.com/svg/svgo/issues/1141#issuecomment-547226332:

Yeah thought svgo was a solid idea when I saw it in gatsby

```
it-20750:engine-theme-sdk howardj$ npm run minify:svg -- --enable=removeDimensions

> @wpmedia/engine-theme-sdk@2.2.3 minify:svg /Users/howardj/sites/engine-theme-sdk
> svgo -f svg "--enable=removeDimensions"

Processing directory 'svg':


arc-placeholder-logo.svg:
Done in 21 ms!
0.762 KiB - 0.4% = 0.759 KiB

arrow-right.svg:
Done in 4 ms!
0.333 KiB - 0.9% = 0.33 KiB

backward.svg:
Done in 2 ms!
0.523 KiB - 0.6% = 0.521 KiB

camera.svg:
Done in 1 ms!
0.539 KiB - 0.5% = 0.536 KiB

calendar.svg:
Done in 4 ms!
0.454 KiB - 0.6% = 0.451 KiB

chart.svg:
Done in 2 ms!
0.781 KiB - 0.4% = 0.778 KiB

bell.svg:
Done in 2 ms!
0.437 KiB - 0.7% = 0.434 KiB

arrow-left.svg:
Done in 1 ms!
0.334 KiB - 0.9% = 0.331 KiB

chevron-left.svg:
Done in 1 ms!
0.266 KiB - 1.1% = 0.263 KiB

chevron-down.svg:
Done in 1 ms!
0.264 KiB - 1.1% = 0.261 KiB

check.svg:
Done in 1 ms!
0.28 KiB - 1% = 0.277 KiB

chevron-right.svg:
Done in 1 ms!
0.266 KiB - 1.1% = 0.263 KiB

close.svg:
Done in 1 ms!
0.424 KiB - 0.7% = 0.421 KiB

cog.svg:
Done in 5 ms!
0.91 KiB - 0.3% = 0.907 KiB

closed-captioning.svg:
Done in 2 ms!
0.721 KiB - 0.4% = 0.718 KiB

chevron-up.svg:
Done in 1 ms!
0.264 KiB - 1.1% = 0.261 KiB

delete.svg:
Done in 2 ms!
0.567 KiB - 0.5% = 0.564 KiB

download.svg:
Done in 2 ms!
0.624 KiB - 0.5% = 0.621 KiB

edit.svg:
Done in 2 ms!
0.598 KiB - 0.5% = 0.595 KiB

ellipsis-h.svg:
Done in 1 ms!
0.412 KiB - 0.7% = 0.409 KiB

ellipsis-v.svg:
Done in 1 ms!
0.409 KiB - 0.7% = 0.406 KiB

comments.svg:
Done in 1 ms!
0.642 KiB - 0.5% = 0.639 KiB

envelope.svg:
Done in 2 ms!
0.581 KiB - 0.5% = 0.578 KiB

envelope-open.svg:
Done in 2 ms!
0.715 KiB - 0.4% = 0.712 KiB

forward.svg:
Done in 1 ms!
0.522 KiB - 0.6% = 0.52 KiB

facebook.svg:
Done in 1 ms!
0.373 KiB - 0.8% = 0.37 KiB

external-link.svg:
Done in 2 ms!
0.524 KiB - 0.6% = 0.521 KiB

facebook-alt.svg:
Done in 1 ms!
0.39 KiB - 0.8% = 0.387 KiB

gift.svg:
Done in 1 ms!
0.635 KiB - 0.5% = 0.632 KiB

full-screen.svg:
Done in 4 ms!
0.641 KiB - 0.5% = 0.638 KiB

grip-vertical.svg:
Done in 2 ms!
0.921 KiB - 0.3% = 0.918 KiB

hamburger-menu.svg:
Done in 1 ms!
0.449 KiB - 0.7% = 0.446 KiB

headphones.svg:
Done in 2 ms!
0.522 KiB - 0.6% = 0.52 KiB

grid.svg:
Done in 2 ms!
1.22 KiB - 0.2% = 1.217 KiB

instagram.svg:
Done in 2 ms!
1.106 KiB - 0.3% = 1.104 KiB

home.svg:
Done in 1 ms!
0.616 KiB - 0.5% = 0.613 KiB

link.svg:
Done in 2 ms!
0.85 KiB - 0.3% = 0.847 KiB

linkedIn-alt.svg:
Done in 1 ms!
0.504 KiB - 0.6% = 0.501 KiB

lock.svg:
Done in 1 ms!
0.37 KiB - 0.8% = 0.367 KiB

linkedin.svg:
Done in 1 ms!
0.394 KiB - 0.7% = 0.391 KiB

medium-logo.svg:
Done in 2 ms!
0.403 KiB - 0% = 0.403 KiB

next.svg:
Done in 1 ms!
0.284 KiB - 1% = 0.281 KiB

pause-circle.svg:
Done in 2 ms!
0.462 KiB - 0.6% = 0.459 KiB

page.svg:
Done in 2 ms!
0.611 KiB - 0.5% = 0.608 KiB

pause.svg:
Done in 2 ms!
0.367 KiB - 0.8% = 0.364 KiB

phone.svg:
Done in 1 ms!
0.37 KiB - 0.8% = 0.367 KiB

pinterest-alt.svg:
Done in 2 ms!
0.817 KiB - 0.4% = 0.814 KiB

play-circle.svg:
Done in 1 ms!
0.319 KiB - 0.9% = 0.316 KiB

pinterest.svg:
Done in 2 ms!
0.602 KiB - 0.5% = 0.599 KiB

plus.svg:
Done in 1 ms!
0.381 KiB - 0.8% = 0.378 KiB

previous.svg:
Done in 1 ms!
0.275 KiB - 1.1% = 0.272 KiB

play.svg:
Done in 1 ms!
0.207 KiB - 1.4% = 0.204 KiB

print.svg:
Done in 2 ms!
0.539 KiB - 0.5% = 0.536 KiB

reddit-alien.svg:
Done in 2 ms!
0.904 KiB - 0% = 0.904 KiB

search.svg:
Done in 2 ms!
0.479 KiB - 0.6% = 0.476 KiB

share.svg:
Done in 2 ms!
0.609 KiB - 0.5% = 0.606 KiB

snapchat-ghost.svg:
Done in 4 ms!
1.445 KiB - 0% = 1.445 KiB

sound-off.svg:
Done in 2 ms!
0.521 KiB - 0.6% = 0.519 KiB

star-half.svg:
Done in 1 ms!
0.54 KiB - 0.5% = 0.537 KiB

shopping-cart.svg:
Done in 2 ms!
0.602 KiB - 0% = 0.602 KiB

tumblr-logo.svg:
Done in 1 ms!
0.435 KiB - 0% = 0.435 KiB

sound-on.svg:
Done in 2 ms!
0.784 KiB - 0.4% = 0.781 KiB

soundcloud.svg:
Done in 5 ms!
2.137 KiB - 0% = 2.137 KiB

twitch.svg:
Done in 1 ms!
0.297 KiB - 1% = 0.294 KiB

star.svg:
Done in 1 ms!
0.36 KiB - 0.8% = 0.357 KiB

twitter-alt.svg:
Done in 2 ms!
0.715 KiB - 0.4% = 0.712 KiB

twitter.svg:
Done in 1 ms!
0.631 KiB - 0.5% = 0.628 KiB

zoom-in.svg:
Done in 2 ms!
0.723 KiB - 0.4% = 0.72 KiB

unlock.svg:
Done in 1 ms!
0.42 KiB - 0.7% = 0.417 KiB

user.svg:
Done in 0 ms!
0.31 KiB - 0.9% = 0.307 KiB

whatsapp.svg:
Done in 1 ms!
0.943 KiB - 0% = 0.943 KiB

zoom-out.svg:
Done in 1 ms!
0.588 KiB - 0.5% = 0.585 KiB

arrow-down.svg:
Done in 1 ms!
0.328 KiB - 0.9% = 0.325 KiB

RSS.svg:
Done in 3 ms!
0.467 KiB - 0.6% = 0.464 KiB

arrow-up.svg:
Done in 1 ms!
0.335 KiB - 0.9% = 0.332 KiB

youtube-alt.svg:
Done in 1 ms!
0.578 KiB - 0.5% = 0.575 KiB

youtube.svg:
Done in 1 ms!
0.43 KiB - 0.7% = 0.427 KiB
it-20750:engine-theme-sdk howardj$

**Review by brentmiller1973 — APPROVED** [review]
Hell yeah!  Nice!

**Comment by JackHowa** [comment]
> At his point I've completely forgotten my reasoning for including the SVGs alongside the icon components.

@jgrosspietsch 🤔 I wonder if I can just remove them in another pass? Just reached out to @kascote

**Pull request merged**